### PR TITLE
Add trackers summary

### PR DESCRIPTION
**Pull request by superpoussin22**
### Describe the Pull Request

```
Add Trackers summary in app score
```

### Checklist for PR

- [X] Run MobSF unit tests (http://your-mobsf-@ip:8000/tests/ or python3 manage.py test).
- [X] Tested Working on Linux, Mac, Windows, and Docker
- [X] Make sure build is passing on your PR. [Build Status](https://travis-ci.com/MobSF/Mobile-Security-Framework-MobSF/pull_requests)
- [X] Coding style (indentation, etc)

### CODE REVIEW

File: templates/static_analysis/android_binary_analysis.html [hunk 1]
```diff
@@ -126,6 +126,10 @@ <h3 class="box-title">App Score</h3>
           </br>
           <span class="label label-danger">VirusTotal Detection</span> <b>{{ VT_RESULT.positives }}/{{ VT_RESULT.total }}</b>
         {% endif %}
+        {% if trackers %}
+          </br>
+          <span class="label label-danger">Trackers Detection</span> <b>{{ trackers.detected_trackers }}/{{ trackers.total_trackers }}</b>
```
Comment (ajinabraham) [inline templates/static_analysis/android_binary_analysis.html]: how about `label-danger` ?

Comment (superpoussin22) [reply templates/static_analysis/android_binary_analysis.html]: @ajinabraham which one sounds the best for You ?

Comment (ajinabraham) [reply templates/static_analysis/android_binary_analysis.html]: sorry` label-warning`

Comment (superpoussin22) [reply templates/static_analysis/android_binary_analysis.html]: done :) and yep better to put only a warning

**Comment by ajinabraham** [comment]
👍 Good to merge!

**Pull request merged**